modules/effects/src/main/scala/dev/profunktor/redis4cats/JNumber.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2018-2020 ProfunKtor
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dev.profunktor.redis4cats
+
+import cats.Applicative
+import cats.implicits._
+
+object JNumber {
+
+  implicit class JLongOps[F[_]](value: F[java.lang.Long])(implicit F: Applicative[F]) {
+    def asScala: F[Long] = value.map(_.longValue)
```

Comment:
I'm generally not opposed to having extra syntax but I don't think it is justified in this case, I'd prefer to keep using `map(x => Long.box(x))` in the interpreter.

Notice that there's also `Double.box`, `Boolean.box`, etc so we would end up having overloaded `asScala` methods, which can be confusing, even to the Scala compiler.